pkg/metrics/otel.go

```diff
@@ -204,38 +204,38 @@ processors:
         operations:
           - action: aggregate_labels
             label_set: [component, errorclass]
-            aggregation_type: sum
+            aggregation_type: max
       - include: reconcile_duration_seconds
         action: update
         operations:
           - action: aggregate_labels
             label_set: [status]
-            aggregation_type: sum
+            aggregation_type: max
       - include: rg_reconcile_duration_seconds
         action: update
         operations:
           - action: aggregate_labels
             label_set: [stallreason]
-            aggregation_type: sum
+            aggregation_type: max
       - include: last_sync_timestamp
         action: update
         operations:
           - action: aggregate_labels
             label_set: [status]
-            aggregation_type: sum
+            aggregation_type: max
       - include: parser_duration_seconds
         action: update
         operations:
           - action: aggregate_labels
             label_set: [status, source, trigger]
-            aggregation_type: sum
+            aggregation_type: max
       - include: pipeline_error_observed
         action: update
         new_name: last_pipeline_error_observed
         operations:
           - action: aggregate_labels
             label_set: [name, component, reconciler]
-            aggregation_type: sum
+            aggregation_type: max
       - include: apply_operations_total
         action: update
         new_name: apply_operations_count
```

Comment:
This is interesting.
Every time this is recorded the value is always 1. so summing will change that.
I don't think this is a problem, as long as we graph by sum in our dashboards over a time period or something...

Reply:
The pipeline error is an odd one iirc. The value of it is not used as counter but more of a boolean status, where non-zero indicates error. Sum probably works for this case.